### ISSUE TITLE
Gear Disabling Not Working

### ISSUE THREAD

**Issue by Kehlun** [modelleicher/FS19_realManualTransmission#44]
hello modeliecher

I love the mod and add it to every mod I use and all the ingame stuff as well. 
I posted a recommendation a few months ago to add sound and you answered that one fast saying its not a priority. logical.
however now I have discovered an issue and that is adding a specific gearbox to a mod. it uses the disable gears code and either the mod doesn't even show up after adding it or it shows up but doesn't allow you to use the disables gears in the ranges. ill paste the gearbox now. example the reverse I cant use in the game with this gearbox

	
	<realManualTransmissionConfigurations>
	
		<realManualTransmissionConfiguration>

			 <realManualTransmission>

				<gears>
					<gear speed="3.9" name="1" />
					<gear speed="6.2" name="2" />
					<gear speed="9.7" name="3" />
					<gear speed="14.7" name="4" />
					<gear speed="22.8" name="5" />
					<gear speed="40.0" name="6" />
				</gears>

				<rangeSet1 powerShift="false" defaultRange="2" >
					<range ratio="0.805" name="L" />
					<range ratio="1.0" name="H" />
					**<range ratio="1.306" name="R" isReverse="true" >
						<disableGears gears="6" disableType="neutral" />**
					</range>
				</rangeSet1>
					
				<rangeSet2 powerShift="false" defaultRange="2" >
					**<range ratio="0.504" name="Lo" >
						<disableGears gears="5 6" disableType="lock" />**
					</range>
					<range ratio="1" name="" />
				</rangeSet2>
				
			</realManualTransmission>

		</realManualTransmissionConfiguration>
		
	</realManualTransmissionConfigurations>

**the part like this is what isn't showing up or usable ingame any ideas to why?**

**Comment by DarcReaver** [comment]
Can confirm, I also tried various combinations of the disable gear feature with the same result.

**Comment by modelleicher** [comment]
Bug appeared during multiplayer rewrite.. need to track down where the issue is but haven't had time yet

**Comment by DarcReaver** [comment]
While we're at it: is there a possibilty to disable ranges when using a regular Forward-N-Reverse reverser? I wanted to map the Maxxum 5150, it has 4 range H-Shift (I-IV) and 4 powershiftable gears. Forward has ranges I-IV and reverse is I-III, with +18% speed on those backward ranges compared to forward.

**Comment by modelleicher** [comment]
So basically disable ranges per reverser?

**Comment by modelleicher** [comment]
So basically disable ranges per reverser?

**Comment by DarcReaver** [comment]
Yes, something like that. If it's easier a true/false statement could also work. Something like "range forwardOnly true/false".
